apps/explorer/src/app/components/truncate/truncate.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+
+const ELLIPSIS = '\u2026';
```

Comment:
praise (non-blocking): nicely done using proper ellipseses? Elipplisi? Elpipsisesses? Falndereseses?

Reply:
Haha! Bouncing it over to @mattrussell36 as this is his work :)